docs/src/tablespaces.md

```diff
@@ -291,21 +291,21 @@ spec:
 
 PostgreSQL allows you to define one or more temporary tablespaces to create
 temporary objects (temporary tables and indexes on temporary tables) when a
-`CREATE` command does not explicitly specify a tablespace, as well as temporary
+`CREATE` command doesn't explicitly specify a tablespace. It also allows you to define temporary
 files for purposes such as sorting large data sets. When no temporary
-tablespace is specified, PostgreSQL uses the default tablespace of a database -
+tablespace is specified, PostgreSQL uses the default tablespace of a database, which is
 currently the main `PGDATA` volume.
 
 When you specify more than one temporary tablespace, PostgreSQL randomly picks
-one the first time a temporary object needs to be created in a transaction,
-then sequentially iterates through the list.
+one the first time a temporary object needs to be created in a transaction.
+Then it sequentially iterates through the list.
 
-Temporary tablespaces work like regular tablespaces, also regarding backups.
+Temporary tablespaces also work like regular tablespaces with regard to backups.
```

Comment:
I think part of the meaning is lost here. The main message is that temporary tablespaces work like regular tablespaces.
The backups part is just for reassurance, not the main take-away.
Not sure how to say it better though...

Reply:
I agree